bridgerelayer/go.mod

```diff
@@ -3,14 +3,16 @@ module github.com/snowfork/polkadot-ethereum/bridgerelayer
 go 1.13
 
 require (
-	github.com/centrifuge/go-substrate-rpc-client v1.1.1-0.20200428140716-e238735648d6
+	github.com/Snowfork/go-substrate-rpc-client v2.0.0-alpha.5.0.20200730202536-d47ecb76adac+incompatible
```

Comment:
Looks like we've got 3x `go-substrate-rpc-client` deps.
1. We're using both `github.com/Snowfork/go-substrate-rpc-client` and `github.com/snowfork/go-substrate-rpc-client`, if possible let's just use one version of the forked library in the Relayer.
2. See if `go mod tidy` removes the centrifuge grpc indirect dependency. If not, we can take a look at removing it in the next milestone by updating dependency imports in `Snowfork/go-substrate-rpc-client`.

Reply:
Nice catch. The `go mod tidy` command didn't work, so I've cut a ticket to track: https://github.com/Snowfork/polkadot-ethereum/issues/43.